### PR TITLE
Add ie11 polyfill package and test with react-button package

### LINKED ISSUE #14668: IE 11 polyfill for CSS variables
This is to add a polyfill for ie11 since it does not support css variables. Main work items are:

- [x] Make a package to put this polyfill script and future ie11 polyfills
- [x] Check in polyfill script to the package
- [x] Test polyfill on component that uses css variables(Button in @fluentui/react-button)
- [x] Write documentation for using the polyfill
- [ ] Repackage (soft fork)

For websites that use hot reloading, pages that need the polyfill script would need to be refreshed, or the polyfill script needs to be put in these pages.

### PR DESCRIPTION
**Pull request by tomi-msft**
#### Pull request checklist

- [x] Addresses an existing issue: Fixes #14668
- [x] Include a change request file using `$ yarn change`

#### Description of changes

(give an overview)

#### Focus areas to test

(optional)

### CODE REVIEW

File: packages/ie11-custom-properties/LICENSE
Comment (ecraig12345) [inline packages/ie11-custom-properties/LICENSE:3]: @dzearing What should we do about the copyright and authorship notice since this is forked from another package?

File: packages/ie11-custom-properties/README.md
Comment (ecraig12345) [inline packages/ie11-custom-properties/README.md:3]: Update this so it makes sense :)

File: packages/ie11-custom-properties/ie11CustomProperties.js
Comment (ecraig12345) [inline packages/ie11-custom-properties/ie11CustomProperties.js:1]: Should probably remove this line, or replace it with something indicating the original source.

File: packages/example-app-base/src/components/CodeSnippet/CodeSnippet.tsx
Comment (ecraig12345) [inline packages/example-app-base/src/components/CodeSnippet/CodeSnippet.tsx:41]: Oh weird, this is not where I thought the error came from--this part is a bit older (1 month) and it really seems like the default FluentTheme ought to be defined...

**Comment by khmakoto** [comment]
Waiting until we figure out licensing to merge this PR.

File: packages/ie11-custom-properties/ie11CustomProperties.js
Comment (tomi-msft) [reply packages/ie11-custom-properties/ie11CustomProperties.js]: I have removed this line

File: packages/ie11-custom-properties/README.md
Comment (ecraig12345) [inline packages/ie11-custom-properties/README.md:3]:
```suggestion
**IE11 CSS variables polyfill**
```

File: packages/ie11-custom-properties/README.md
Comment (ecraig12345) [inline packages/ie11-custom-properties/README.md:5]: Would be good to specify what this is a polyfill for. There are lots of other types of polyfills besides for CSS variables (like for adding Promise, newer array methods, etc).

Comment (tomi-msft) [reply packages/ie11-custom-properties/README.md]: Added more info to the README

File: change/@fluentui-ie11-custom-properties-2020-08-27-12-13-14-ie11-polyfill.json
Comment (ecraig12345) [inline change/@fluentui-ie11-custom-properties-2020-08-27-12-13-14-ie11-polyfill.json:2]: Delete this file

File: packages/ie11-polyfills/README.md
Comment (ecraig12345) [inline packages/ie11-polyfills/README.md:3]:
```suggestion
**IE11 polyfills for Fluent UI React**
```

File: packages/ie11-polyfills/README.md
Comment (ecraig12345) [inline packages/ie11-polyfills/README.md:13]: Some nitpicky suggestions
```suggestion
Package to hold IE11 polyfill that allows webpages that use CSS variables for styling to render correctly in IE11. It will also hold any future IE11 polyfills

To use the polyfills, just import the package:

```js
import '@fluentui/ie11-polyfills';
```

If you are using any websites with hot reloading, you will have to either put the polyfill on each page that uses CSS variables, or refresh the page for the script to take effect.
```

File: packages/ie11-polyfills/index.js
Comment (ecraig12345) [inline packages/ie11-polyfills/index.js:1]: If we're going to use ES6 export syntax like this, the package will need to use TS to transpile (not everything can handle ES6 exports). Also, it would be better to just do `import 'ie11-custom-properties'` since that package relies on side effects rather than exporting anything.

The other option would be to keep it as JS and change this line to `require('ie11-custom-properties')`.

I can see arguments for either keeping as JS (mainly that there's no build cost) or converting to TS (which would probably be needed if we add more polyfills later). Unless someone else has strong feelings I think we should stick with JS for now.

File: packages/ie11-polyfills/just.config.ts
Comment (ecraig12345) [inline packages/ie11-polyfills/just.config.ts:1]: Assuming we stick with JS (not TS) delete this file

File: packages/ie11-polyfills/package.json
Comment (ecraig12345) [inline packages/ie11-polyfills/package.json:16]: Delete this if we're using JS

File: packages/ie11-polyfills/index.js
Comment (tomi-msft) [reply packages/ie11-polyfills/index.js:1]: Will stick with JS for now

File: change/@uifabric-experiments-2020-08-27-14-11-31-ie11-polyfill.json
Comment (ecraig12345) [inline change/@uifabric-experiments-2020-08-27-14-11-31-ie11-polyfill.json:2]: Remove this file (I think it was fixed in master)

**Review by ecraig12345 — APPROVED** [review]
Looks good once you delete the extra change file.

**Pull request merged**
